RELEASE/scripts/autoscend/quests/optional.ash

```diff
@@ -365,6 +365,14 @@ boolean LX_guildUnlock()
 	string pref;
 	location loc = $location[None];
 	item goal = $item[none];
+	if(my_primestat() == $stat[Moxie] && auto_haveTearawayPants())
```

Comment:
given that unlocking the Moxie guild is free with tearaway pants, I feel like you should do it even if `auto_skipUnlockGuild` is true, and unlock Whitey's Grove early so you can spend spare wishes on bird rib / lion oil.

That is, treat `auto_skipUnlockGuild` as "don't spend any adventures unlocking the guild" instead of "never unlock the guild".